phishing/url-analysis-advanced.md

```diff
@@ -10,7 +10,7 @@
 ## Lesson goals
 - Obfuscation of URLs and Pages
 - Clickjacking
-- Open Source INTelligence (OSINT) gathering
+- Open Source Intelligence (OSINT) gathering
```

Comment:
Use capitalization as it was - since this is calling out the relevant letters in the acronym